tools/find-inactive-tsc.mjs

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env node
+
+// Identify inactive TSC members.
+
+// From the TSC Charter:
+//   A TSC member is automatically removed from the TSC if, during a 3-month
+//   period, all of the following are true:
+//     * They attend fewer than 25% of the regularly scheduled meetings.
+//     * They do not participate in any TSC votes.
+
+import cp from 'node:child_process';
+import fs from 'node:fs';
+import path from 'node:path';
+import readline from 'node:readline';
+
+const SINCE = +process.argv[2] || '3 months ago';
+
+async function runGitCommand(cmd, options = {}) {
+  const childProcess = cp.spawn('/bin/sh', ['-c', cmd], {
+    cwd: options.cwd ?? new URL('..', import.meta.url),
+    encoding: 'utf8',
+    stdio: ['inherit', 'pipe', 'inherit'],
+  });
+  const lines = readline.createInterface({
+    input: childProcess.stdout,
+  });
+  const errorHandler = new Promise(
+    (_, reject) => childProcess.on('error', reject)
+  );
+  let returnValue = options.mapFn ? new Set() : '';
+  await Promise.race([errorHandler, Promise.resolve()]);
+  // If no mapFn, return the value. If there is a mapFn, use it to make a Set to
+  // return.
+  for await (const line of lines) {
+    await Promise.race([errorHandler, Promise.resolve()]);
+    if (options.mapFn) {
+      const val = options.mapFn(line);
+      if (val) {
+        returnValue.add(val);
+      }
+    } else {
+      returnValue += line;
+    }
+  }
+  return Promise.race([errorHandler, Promise.resolve(returnValue)]);
+}
+
+async function getTscFromReadme() {
+  const readmeText = readline.createInterface({
+    input: fs.createReadStream(new URL('../README.md', import.meta.url)),
+    crlfDelay: Infinity,
+  });
+  const returnedArray = [];
+  let foundTscHeading = false;
+  for await (const line of readmeText) {
+    // If we've found the TSC heading already, stop processing at the next
+    // heading.
+    if (foundTscHeading && line.startsWith('#')) {
+      break;
+    }
+
+    const isTsc = foundTscHeading && line.length;
+
+    if (line === '### TSC (Technical Steering Committee)') {
+      foundTscHeading = true;
+    }
+    if (line.startsWith('* ') && isTsc) {
+      const handle = line.match(/^\* \[([^]+)]/)[1];
+      returnedArray.push(handle);
+    }
+  }
+
+  if (!foundTscHeading) {
+    throw new Error('Could not find TSC section of README');
+  }
+
+  return returnedArray;
+}
+
+async function getAttendance(tscMembers, meetings) {
+  const attendance = {};
+  for (const member of tscMembers) {
+    attendance[member] = 0;
+  }
```

Comment:
I fond the following more "to the point", but feel free to disagree and disregard.
```suggestion
  const attendance = Object.fromEntries(tscMembers.map(member => [member, 0]));
```

Reply:
I find the for loop more readable, so I'll take you up on the "feel free to...disregard" but if others feel strongly that using `fromEntries()` is the way to go here, I'll shrug and do it. For now, hitting the _Resolve conversation_ button.